README.md

```diff
@@ -61,13 +61,12 @@ We also have a [shared calendar](https://zoom-lfx.platform.linuxfoundation.org/m
 
 All of our meeting recordings are available for review on our YouTube [channel](https://www.youtube.com/@OSCAL-Compass).
 
-##### Email
+### Email
 
 Google Group: [oscal-compass](https://groups.google.com/g/oscal-compass)\
 oscal-compass@googlegroups.com
 
-
-##### Chat anytime
+### Chat anytime
```

Comment:
Maybe just me, but when I follow link to chat I am prompted to login. Fine. I choose email and get code to enter. Fine. Then I arrive at slack and am prompted to sign in to kubernetes. This is correct?

Reply:
I tried to reproduce, but I am always directed to CNCF Slack.